### PR TITLE
Create stubs to match implementation of Bureau Details API

### PR DESCRIPTION
**Pull request by cnorthwood**

### CODE REVIEW

File: swagger/v0/swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        '501':
+          description: is a deprecated API which has not been reimplemented
+  "/api/v0/member/id/{id}":
+    get:
+      summary: Shows full details for a member
+      parameters:
+      - name: id
+        in: path
+        description: "{id} can be a serial number, eg 100002, or a membership number,
+          eg 70/0023"
```
Comment (danieldevlewis) [inline swagger/v0/swagger.yaml:57]: I suspect the `00/0000` form will be slightly problematic as in url terms it is two path parts

**Review by danieldevlewis — COMMENTED** [review]
This all looks correct to me 👍

File: swagger/v0/swagger.yaml (hunk 1 above)
Comment (cnorthwood) [reply swagger/v0/swagger.yaml:57]: Ruby has a syntax that you can use to glob this (the test passes with the dummy URL - in that it currently returns 501 so I'm confident routing is set up correctly)

Comment (danieldevlewis) [reply swagger/v0/swagger.yaml:57]: 👍

**Pull request merged**